pants-plugins/api_spec/rules.py

```diff
@@ -144,87 +91,59 @@ async def generate_api_spec_via_fmt(
             pex,
             argv=(
                 "--config-file",
-                "conf/st2.dev.conf",
+                subsystem.config_file,
             ),
             input_digest=input_digest,
             description="Regenerating openapi.yaml api spec",
             level=LogLevel.DEBUG,
         ),
     )
 
-    contents = [
-        FileContent(
-            f"{field_set.address.spec_path}/{field_set.source.value}",
-            result.stdout,
-        )
-        for field_set in request.field_sets
-    ]
+    contents = [FileContent(file, result.stdout) for file in request.files]
 
     output_digest = await Get(Digest, CreateDigest(contents))
     output_snapshot = await Get(Snapshot, Digest, output_digest)
-    # TODO: Drop result.stdout since we already wrote it to a file?
-    return FmtResult.create(request, result, output_snapshot, strip_chroot_path=True)
+
+    return FmtResult(
+        input=request.snapshot,
+        output=output_snapshot,
+        # Drop result.stdout since we already wrote it to a file
+        stdout="",
+        stderr=strip_v2_chroot_path(result.stderr),
+        tool_name=request.tool_name,
+    )
 
 
 @rule(
     desc="Validate openapi.yaml with st2-validate-api-spec",
     level=LogLevel.DEBUG,
 )
 async def validate_api_spec(
-    request: ValidateAPISpecRequest,
-) -> LintResults:
+    request: ValidateAPISpecRequest.Batch,
+    subsystem: ValidateApiSpec,
+) -> LintResult:
     # There will only be one target+field_set, but we iterate
```

Comment:
Does this comment make sense now we have removed this chunk of code?

Reply:
Hmm. Yup. This is out-of-date. I'll drop the comment.